plutus-e2e-tests/test/Spec/Builtins/SECP256k1.hs

```diff
@@ -27,6 +27,7 @@ tests = testGroup "SECP256k1"
   [ testProperty "unable to use SECP256k1 builtins in Alonzo PV6" (verifySchnorrAndEcdsa testnetOptionsAlonzo6)
   , testProperty "unable to use SECP256k1 builtins in Babbage PV7" (verifySchnorrAndEcdsa testnetOptionsBabbage7)
   , testProperty "can use SECP256k1 builtins in Babbage PV8" (verifySchnorrAndEcdsa testnetOptionsBabbage8)
+  --, testProperty "can use SECP256k1 builtins in Babbage PV8 (on preview testnet)" (verifySchnorrAndEcdsa localNodeOptionsPreview)
```

Comment:
This is the line to be commented in when wanting to run on preview?

Reply:
Yeah, just added that to try and make it clearer what to do. Can add inline comment or remove if it's not useful.

Reply:
Definitely keep the line and add a comment, as otherwise anyone else would wonder where to begin when wanting to test on the public testnets, this line shows the ropes. :)